supportedBackends/jes/src/main/scala/cromwell/backend/impl/jes/JesJobExecutionActor.scala

```diff
@@ -95,4 +98,12 @@ case class JesJobExecutionActor(override val jobDescriptor: BackendJobDescriptor
   }
 
   override def abort(): Unit = {}
+
+  // Supervision strategy: if the JABJEA throws an exception, stop the actor and fail the job.
+  def jobFailingDecider: Decider = {
+    case e: Exception =>
+      completionPromise.failure(new RuntimeException("JesAsyncBackendJobExecutionActor failed and didn't catch its exception.", e))
```

Comment:
ToL: `Promise.tryFailure` is more forgiving. I do not know if it applies here for the JES backend Promises/Futures/Actors, but `Promise.failure` [throws exceptions](http://docs.scala-lang.org/overviews/core/futures.html) if the promise is already completed for some unexpected reason (the scala `Promise` is completed in some other thread, then an exception escapes elsewhere in the akka supervised `Actor`).

"ToL" because I don't know off the top of my head what happens if this `Decider` throws an exception.